### DISCUSSION TITLE
976D6A97-8518-950E-7081-80707338BFAE

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#194] [General]
# Mira

Mira is a binary star system comprising of a red giant and a white dwarf about 443.5 light-years away from the constellation Cetus. It is one of the first variable stars ever discovered, where the apparent magnitude changes over a period of time as seen from Earth.

https://celestia.mobi/addon?item=976D6A97-8518-950E-7081-80707338BFAE

**Comment by SiriusBislifeless** [comment]
Nevermind, I think this is the oldest one

**Reply by DaveBowman2001** [reply]
Reading your comment from the [other addon](https://celestia.mobi/addon?item=FC1DD4F4-60E5-B4CB-0C0C-8CF030F8C005), nope this isn't the oldest addon created. There are lots more created as early as 2001–2004 (mostly nebulae/spacecraft/fictional addons), some were either ported directly to become available for download in modern versions of Celestia (including mobile) or replaced with newer versions

**Comment by denial9999** [comment]
I like this star

**Comment by maratmuratbekov073-droid** [comment]
Wait how could a star exist with a colorful accretion disk with a red giant it’s maybe very rare in the observable universe

**Reply by DaveBowman2001** [reply]
The accretion disk's colors were only mostly artistic choices made by the original author of the addon. We do not know what its actual colors look like
